addon/utils/module-entries.js

```diff
@@ -0,0 +1,7 @@
+/*globals requirejs */
+
+if (typeof requirejs.entries === 'undefined') {
+  requirejs.entries = requirejs._eak_seen;
+}
+
+export default requirejs.entries;
```

Comment:
i would prefer if we didn't share the `entries` directory, rather exposed accessors. Sharing the raw structure gives me the heebee jeebees.

* `has`
* `get`